CMakeLists.txt

```diff
@@ -35,6 +34,19 @@ endif()
 set(TARGET_DEVICE "PINETIME" CACHE STRING "Target device")
 set_property(CACHE TARGET_DEVICE PROPERTY STRINGS PINETIME MOY-TFK5 MOY-TIN5 MOY-TON5 MOY-UNK)
 
+option(ENABLE_APP_STOPWATCH "Enable the Stopwatch application" True)
```

Comment:
We can move those options and the Apps.h.in file into its own directory, create a CMake header only library target and link that to everywhere we need it. Because it is in its own folder I can easily include it in InfiniSim afterwards. Like we do for the fonts and images

Reply:
I'll have a look at this!

Reply:
Done  in [c200816](https://github.com/InfiniTimeOrg/InfiniTime/pull/1928/commits/c200816416e09b890237625ab27d947904843eae) and integrated in InfiniSim here : https://github.com/InfiniTimeOrg/InfiniSim/pull/134